src/server/transaction.h

```diff
@@ -233,7 +233,7 @@ class Transaction {
   void StartMultiGlobal(DbIndex dbid);
 
   // Start multi in LOCK_AHEAD mode with given keys.
-  void StartMultiLockedAhead(DbIndex dbid, CmdArgList keys);
+  void StartMultiLockedAhead(DbIndex dbid, CmdArgVec keys);
```

Comment:
Best I can do: pass owned vector. For EXEC it's free, for EVAL we copy (but I removed a hashset in this PR so we still have gains! 😆 )

Reply:
I agree. I think we gain a lot more than this. For keys that are longer than small object strings, we save tons of allocations when locking.